media-libs/libopenshot-audio/libopenshot-audio-0.0.6.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Audio library used by OpenShot"
+HOMEPAGE="http://www.openshot.org/ https://launchpad.net/libopenshot"
+SRC_URI="https://launchpad.net/libopenshot/0.1/0.1.0/+download/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
```

Comment:
Checking ```/usr/lib64/libopenshot-audio.so.0.0.6``` with ```scanelf -n``` reveals a couple of potential missed dependencies - ```x11-libs/libX11``` and ```x11-libs/libXext```.

Also consider sorting the dependency list.